analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UriShouldNotBeHardcoded.CSharp11.cs

```diff
@@ -17,22 +17,29 @@ public void RawStrings()
         var unixPath2 = """~/blah/blah/blah.txt"""; // Compliant
         var unixPath3 = """~\\blah\\blah\\blah.txt"""; // Compliant
         var windowsPathStartingWithVariable = """%AppData%\\Adobe""";
-
-        var webChemin = """http://www.mywebsite.com"""; // FN
```

Comment:
Test cases on line 21 and 22 are redundant: raw string literals test cases are reported above, while the non-English variable names FNs are documented in the base test cases file.